frontend/src/styles/sidebar.scss

```diff
@@ -1,44 +1,51 @@
-.Sidebar {
-    position: fixed;
-    height: 100%;
-    width: $sidebar-closed-width;
-    background-color: $bp-blue;
-    color: $background-base;
- }
-  
-  .SidebarList {
-    height: auto;
-    width: 100%;
-    padding: 0;
- }
-  
-  .SidebarList .row {
-    height: 60px;
-    width: 100%;
-    list-style-type: none;
-    margin: 0;
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    align-items: center;
-    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-  }
-  
-  .SidebarList .row:hover {
-    cursor: pointer;
-    background-color: rgb(211, 211, 211);
-  }
-  
-  .SidebarList #active {
-    background-color: tomato;
-  }
-  
-  .row #icon {
-    flex: 30%;
+.sidebar {
+  height: 100%;
+  width: 100%;
+  background-color: $bp-blue;
+  color: $bp-white;
+}
+
+.sidebarList {
+  height: auto;
+  width: 100%;
+  padding: 0;
+}
+
+.sidebarList .row {
+  height: 80px;
+  width: 100%;
+  list-style-type: none;
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  font-family: $basic-font;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 38px;
+}
+
+.sidebarList .row:hover {
+  cursor: pointer;
+  background: rgb(16, 85, 97);
```

Comment:
Generalnie jak już się tworzy plik ze zmiennymi, to właśnie po to, żeby w "zwykłych" (s)cssach nie było magicznych pikseli i innych wyrażeń typu rbg(16, 85, 97). To nie jest jakiś kluczowy element, ale łatwiej się potem ogarnąć w całości :)

Reply:
Część związaną z kolorami poprawię w commicie zaraz, część związaną z czcionką - na razie zostawiam, bo ten segment i tak jest do przerobienia. Znalazłem już odpowiedź, jak zaciągnąć to niedziałające Roboto, ale będzie to wymagało trochę roboty i na razie nie jest kluczowe

Reply:
A, widzę, że już zostało popawione.
Dzięki :)